include/fuse.h

```diff
@@ -1004,6 +1004,17 @@ int fuse_getgroups(int size, gid_t list[]);
 int fuse_interrupted(void);
 
 /**
+ * Invalidates cache for the given path.
+ *
+ * This calls fuse_lowlevel_notify_inval_inode internally.
+ * This only does something for nodes which are present in
+ * the hashes; it is a no-op for paths to forgotten nodes.
+ *
+ * @return 0 on success, negative error value otherwise.
```

Comment:
I'm still not seeing why the info whether there was a cache entry to be invalidated would be useful. From the point of view of our file system's client, it would make calling `fuse_invalidate_path` more cumbersome because on an error it would have to check and ignore `ENOENT` errors (or whatever we choose the error to be; I still think `ENOENT` would be misleading, perhaps a more obscure errno, such as `ENOKEY`, would be in order). I see your point about hiding info, @trapexit, but IMHO we should favor making the API easier to use in this particular case.

Or should we go for yet another solution and return `1` if there was an entry to invalidate, `0` if there was not, and `-1` on errors? This would make the usage a bit more convenient in that checking for errors would only require checking `fuse_invalidate_path(...) < 0` or `!= -1`.

@Nikratio, what are your thoughts on this?